interp/interp.go

```diff
@@ -124,13 +124,14 @@ type opt struct {
 	cfgDot bool // display CFG graph (debug)
 	// dotCmd is the command to process the dot graph produced when astDot and/or
 	// cfgDot is enabled. It defaults to 'dot -Tdot -o <filename>.dot'.
-	dotCmd   string
-	noRun    bool          // compile, but do not run
-	fastChan bool          // disable cancellable chan operations
-	context  build.Context // build context: GOPATH, build constraints
-	stdin    io.Reader     // standard input
-	stdout   io.Writer     // standard output
-	stderr   io.Writer     // standard error
+	dotCmd      string
+	noRun       bool          // compile, but do not run
+	fastChan    bool          // disable cancellable chan operations
+	context     build.Context // build context: GOPATH, build constraints
+	stdin       io.Reader     // standard input
+	stdout      io.Writer     // standard output
+	stderr      io.Writer     // standard error
+	allowRedecl bool          // allow redeclaration
```

Comment:
no need for allowRedecl, see my comment above gta.go:236

Reply:
👍